tox.ini

```
@@ -12,7 +12,7 @@ extras =
   mysql: mysql
 deps =
   django42: Django>=4.0,<4.2
-  django50: Django>=5.0,<5.1
+  django50: Django>=5.0
```

`django50` debería seguir testeando 5.0.

```suggestion
  django50: Django>=5.0,<5.1
  django51: Django>=5.1,<5.2
```